src/types.ts

```diff
@@ -1518,6 +1518,13 @@ export type AppSettings = {
     p12_cert?: string;
     team_id?: string;
   };
+  async_moderation_config?: {
```

Comment:
isn't it supposed be in response type too ?

Reply:
yes it is. fixed now.